src/components/SelectionList/BaseSelectionList.js

```diff
@@ -261,7 +259,7 @@ function BaseSelectionList({
                 item={item}
                 isFocused={isFocused}
                 isDisabled={isDisabled}
-                onSelectRow={() => selectRow(item, index)}
+                onSelectRow={() => selectRow(item)}
```

Comment:
I'm seeing that `RadioListItem` calls the `onSelectRow` passing the item, I think you can change to

onSelectRow={selectRow}